client/styles/components/_preferences.scss

```diff
@@ -105,6 +129,59 @@
 	width: #{44 / $base-font-size}rem;
 }
 
+.preference__subheading1 {
```

Comment:
I thought of using the same class initially. But I could not find a way to position the elements correctly. I will try finding some other way to position them.